adagucserverEC/LayerTypeLiveUpdate/LayerTypeLiveUpdate.cpp

```diff
@@ -14,8 +14,11 @@ int layerTypeLiveUpdateConfigureDimensionsInDataSource(CDataSource *dataSource)
     requiredDim->value = "2020-01-02T00:00:00Z";
     dataSource->requiredDims.push_back(requiredDim);
   }
-  dataSource->addStep("", NULL);
-  dataSource->getCDFDims()->addDimension("none", "0", 0);
+
+  // // Add step to empty file
+  // dataSource->addStep("", NULL);
```

Comment:
I moved this code, so we no longer need it here. 
The reason why I moved it is because in the case of the solar terminator we still use files, so this empty step does not work.